src/App.jsx

```diff
@@ -22,32 +22,22 @@ const App = () => {
   }, [darkMode]);
 
   return (
-    <div className="w-100 h-100">
+    <div className="w-screen min-h-screen bg-white dark:bg-gray-700">
```

Comment:
I think the control of the background color can remain in the `Background.jsx` component since that is one of the main purposes of the Component. I believe it is just changing this classes `bg-white dark:bg-gray-700` to the other component